index.html

```diff
@@ -267,7 +267,7 @@ <h2>
         `HID` interface
```

Comment:
Please add an "Extension to the `WorkerNavigator` interface" section that's exactly the same as the above but has `WorkerNavigator` instead of `Navigator` and [Exposed=ServiceWorker, SecureContext]

Reply:
Done.